pkgs/applications/networking/cluster/talosctl/default.nix

```diff
@@ -1,18 +1,23 @@
-{ lib, buildGoModule, fetchFromGitHub }:
-
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+let
+  version = "0.14.2";
+  sha256 = "sha256-sQtry94T5cDO+836D/p/8ptQi3WYKDBLr1QZyEXdLQI=";
+  vendorSha256 = "sha256-cd2iNMxWmkSWqqkPLYocUG+fCUXoeUXEuGQxjUWQnXk=";
+  pkgsVersion = "0.9.0-4-gc875fbe";
+  extrasVersion = "0.7.0-2-gb4c9d21";
+in
 buildGoModule rec {
   pname = "talosctl";
-  version = "0.14.1";
+  inherit version vendorSha256;
+  # nixpkgs-update: no auto update
 
   src = fetchFromGitHub {
     owner = "talos-systems";
     repo = "talos";
     rev = "v${version}";
-    sha256 = "sha256-JeZ+Q6LTDJtoxfu4mJNc3wv3Y6OPcIUvgnozj9mWwLw=";
+    inherit sha256;
   };
 
-  vendorSha256 = "sha256-ujbEWvcNJJOUegVgAGEPwYF02TiqD1lZELvqc/Gmb4A=";
-
   # look for GO_LDFLAGS getting set in the Makefile
```

Comment:
Can you move this comment above the `version` line? This was meant as a helper to people bumping to package, to know where to look for.

Reply:
Done.